src/Language/TypeScript/Resolution.hs

```diff
@@ -59,7 +59,6 @@ resolveRelativePath :: ( Has (Modules address value) sig m
                     -> Evaluator term address value m M.ModulePath
 resolveRelativePath relImportPath exts = do
   M.ModuleInfo{..} <- currentModule
-  -- JZ:TODO: shall refactor it later.
   let relRootDir = takeDirectory (Path.toString modulePath)
```

Comment:
I think we can refactor relRootDir and some following variables as pathtype.  But this PR is big enough.  I would like to refactor it in some other small PRs.

Reply:
Sounds good to me!